dev/io.openliberty.jakarta.tags.pages.3.0.internal/.settings/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,276 @@
+#Ant properties
```

Comment:
File seems awfully long.  It may have been messed up by an IDE? Can you double check it?